stories/test_data/fixtures.ts

```diff
@@ -1,4 +1,4 @@
-// (C) 2007-2018 GoodData Corporation
+// (C) 2007-2019 GoodData Corporation
 import { range, cloneDeep } from "lodash";
 import { VisualizationObject } from "@gooddata/typings";
 
```

Comment:
We should revert this file, BarChart.tsx and sorts.ts

Reply:
it is controlled by server, we can't do any thing :(